pkg/persistence/dgraph/service.go

```diff
@@ -39,9 +41,13 @@ type Service struct {
 	Interacts []*Service `json:"interacts,omitempty"`
 }
 
+// PersistService create a new node in the Dgraph or if it exists updates the original node
 func PersistService(service api_v1.Service) error {
 	xid := service.Namespace + ":" + service.Name
-	uid, _ := GetUId(Client, xid, IsService)
+	uid, err := GetUID(Client, xid, IsService)
+	if err != nil {
```

Comment:
no need for err check, GetUID returns uid as "" if err is not nil

Reply:
Same as above.